src/OSPSuite.SimModel/src/TableFormulaWithXArgument.cpp

```diff
@@ -117,7 +117,12 @@ void TableFormulaWithXArgument::DE_Jacobian (double * * jacobian, const double *
 	if (_XArgumentObject->IsConstant(forCurrentRunOnly))
 		return;
 
-	throw ErrorData(ErrorData::ED_ERROR, ERROR_SOURCE, "Table formula with X argument does not support time dependent argument objects");
+	//TODO if X-argument is time dependent but not ODE-variables-dependent, 
+	//     everything is OK
+	// If it's ODE-variables-dependent: either an exception must be thrown or
+	// the Jacobian-Calculation must take it into account
+
```

Comment:
Don't really think we will get this case in the future; but who knows.
So a TODO for the future.

For 7.4(?) I will try to switch from "on-the-fly" Jacobians to precalculated Jacobian functions (which are currently used by C++-ModelExport). This TODO will become part of that task.

Reply:
So the fix is to not thrown an exception?
Can't we check that a parameter only depends on time?

Reply:
For today: yes
Most are for CPP-Model-Export.
Matlab-Export might also become a problem now - will check it in the next days

Reply:
>Can't we check that a parameter only depends on time?

Sure. But not for this release